docs/quick-answers/linode-platform/add-caa-dns-records/index.md

```diff
@@ -6,52 +6,51 @@ description: 'This quick answer shows how to create CAA records for domains and
+     -  **issue** - Authorize the certificate authority entered in the *Value* field further below to issue TLS certificates for your site.
 
-     -  *issuewild* - Same as above, with the exception that you were issued a wildcard certificate.
+     -  **issuewild** - Same as above, with the exception that you were issued a wildcard certificate.
 
-     -  *iodef* - URL where your CA can report security policy violations to you concerning certificate issue requests.
+     -  **iodef** - URL where your CA can report security policy violations to you concerning certificate issue requests.
 
-    **Value**: If the *issue* or *issuewild* tag was selected above, then the value field takes the domain of your certificate issuer (for example: `letsencrypt.org`). If the *iodef* tag was selected, the value field takes a contact or submission URL (`http` or `mailto`).
+    **Value**: If the **issue** or **issuewild** tag was selected above, then the **Value** field takes the domain of your certificate issuer (for example: `letsencrypt.org`). If the **iodef** tag was selected, the **Value** field takes a contact or submission URL (`http` or `mailto`).
 
-    **TTL (Time to Live)**: Time in seconds that your new CAA record will be cached by Linode's DNS name servers before being refreshed. The *Default* selection's TTL is 300 seconds, which is fine for most cases. You can use `dig` to view the remaining time your DNS records will be cached until refreshed. Replace *linode.com* with your site's domain or subdomain in the command below:
+    **TTL (Time to Live)**: Time in seconds that your new CAA record will be cached by Linode's name servers before being refreshed. The *Default* selection's TTL is 300 seconds, which is fine for most cases. You can use the [`dig` command](/docs/networking/dns/use-dig-to-perform-manual-dns-queries/)  to view the remaining time your DNS records will be cached until refreshed. Replace *linode.com* with your site's domain or subdomain in the command below:
 
         root@debian:~# dig +nocmd +noall +answer example.com
         example.com.     167 IN  A   203.0.113.1
 
-5.  Select **Save Changes** when finished. The CAA record should be fully propagated within the TTL duration.
+1.  Click the **Save** button when finished. The CAA record should be fully propagated within the TTL duration.
 
 
 ## Add Multiple CAA Records
 
 Multiple CAA records must be added individually. If your site `example.com` was issued a TLS certificate by Let's Encrypt, but your subdomain `store.example.com` uses a Symantec certificate, you would need two different CAA records. A reporting URL for the *iodef* tag would also need its own record. Those three would look something like this:
 
-![Multiple CAA records](multiple-caa-records.png)
+[![Multiple CAA records](multiple-caa-records.png "Multiple CAA records")](multiple-caa-records.png)
```

Comment:
Making sure the image has both alt and title text,  and wrapping it in a link to itself so someone can zoom in on it if need be